csi.proto

```diff
@@ -945,6 +993,17 @@ message ControllerServiceCapability {
       // Indicates the SP supports the
       // ListVolumesResponse.entry.published_nodes field
       LIST_VOLUMES_PUBLISHED_NODES = 10;
+
+      // Indicates the SP supports the
+      // ListVolumesResponse.entry.volume_health field
+      LIST_VOLUMES_VOLUME_HEALTH = 11 [(alpha_enum_value) = true];
```

Comment:
Should we consistently use volume condition (i.e., `LIST_VOLUMES_VOLUME_CONDITION`) instead given we change the message name?

Reply:
Done